packages/text-input/src/styles.tsx

```diff
@@ -65,6 +65,7 @@ export const getTextInputStyle = (appearance?: TextInputAppearanceType) => (_: {
         backgroundColor: 'transparent',
         boxShadow: colors.input.outline.boxShadow.default,
         ':focus': {
+          color: colors.text.intense,
```

Comment:
Why have you added this?

Reply:
Because this color is on the design, but it should be in the flow-ui, so I will remove it.